app/views/qsets/_question_page_header_small_screen.html.erb

```diff
@@ -18,3 +18,8 @@
     <%= render "all_mine_other_filter" %>
   </div>
 </div>
+<div class="row center">
+  <div class="col-xs-12" id="top-padding">
+    <a href="#" class="btn btn-primary btn-large quiz-all" id="quizAllButtonID"> Quiz All </a>
```

Comment:
As we discussed in one of our meetings, we can't have two elements with the same ID on the page. Looks like this is only used [to enable/disable the button](https://github.com/penzance/askup/pull/85/files#diff-e200cb0d5231dea45b8396296ecfe2c2R97), which can use the `.quiz-all` selector instead.